sovrin_client/agent/endpoint.py

```diff
@@ -57,13 +70,26 @@ def transmitToClients(self, msg: Any, remoteNames: List[str]):
     # TODO: Rename method
     def baseMsgHandler(self, msg):
         logger.debug("Got {}".format(msg))
-        self.msgHandler(msg)
+        self._msgHandler(msg)
 
     def connectTo(self, ha):
-        remote = self.findInRemotesByHA(ha)
+        remote = self.stack.findInRemotesByHA(ha)
         if not remote:
-            remote = RemoteEstate(stack=self, ha=ha)
-            self.addRemote(remote)
+            remote = RemoteEstate(stack=self.stack, ha=ha)
+            self.stack.addRemote(remote)
             # updates the store time so the join timer is accurate
-            self.updateStamp()
-            self.join(uid=remote.uid, cascade=True, timeout=30)
+            self.stack.updateStamp()
+            self.stack.join(uid=remote.uid, cascade=True, timeout=30)
+
+    def isConnectedTo(self, name=None, ha=None):
+        return self.stack.isConnectedTo(name=name, ha=ha)
```

Comment:
`isConnectedTo`, `getRemote`, `transmit`, are simple pass-throughs, more reasons to have an "is a" relationship